### ISSUE TITLE
Installing Test::AutoMock failed

### ISSUE THREAD

**Issue by hiratara** [hiratara/p5-Test-LazyMock#9]
```
#   Failed test at t/tie_hash.t line 33.
#          got: '1/8'
#     expected: '2'
# Looks like you failed 1 test of 22.
t/tie_hash.t ...............
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/22 subtests
```

```
$ perl --version

This is perl 5, version 24, subversion 1 (v5.24.1) built for x86_64-linux
(with 1 registered patch, see perl -V for more detail)

Copyright 1987-2017, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.
```

**Comment by andk** [comment]
It fails for all perls < 5.26, as you can see in the matrix: http://matrix.cpantesters.org/?dist=Test-AutoMock%200.01

See manpage perl5260delta, section:

```
=head2 C<scalar(%hash)> return signature changed

The value returned for C<scalar(%hash)> will no longer show information about
the buckets allocated in the hash.  It will simply return the count of used
keys.  It is thus equivalent to C<0+keys(%hash)>.

A form of backward compatibility is provided via
L<C<Hash::Util::bucket_ratio()>|Hash::Util/bucket_ratio> which provides
the same behavior as
C<scalar(%hash)> provided in Perl 5.24 and earlier.
```

**Comment by hiratara** [comment]
I fixed this issue by #10.
